packages/commonwealth/client/scripts/views/pages/view_thread/poll_cards.tsx

```diff
@@ -32,8 +33,7 @@ export class ThreadPollEditorCard
           thread?
         </CWText>
         <CWButton
-          disabled={!!thread.offchainVotingEndsAt}
```

Comment:
What happened to this logic?

Reply:
`offchainVotingEndsAt` doesn't correspond to anything I can find. No hits in the codebase besides this one.

Reply:
Sure -- but `votingEndTime` still does.